### PR TITLE
feat: KPI-173 order columns by comparison

### PR DESCRIPTION
**Pull request by FreddyJR1995**
## Jira Link

[Go to Jira](https://kpinetwork.atlassian.net/browse/KPI-173)

## Types of changes

- [ ] Hotfix (Bug detected)
- [ x] Feature (Issue from an Active Sprint)

## Checklist:

- [ x] Unit Test.
- [ x] Coverage.

### CODE REVIEW

File: src/views/ComparisonView/ComparisonView.jsx [hunk 1]
```diff
@@ -61,6 +69,10 @@ export function ComparisonView ({ companyComparison, peersComparison, isLoading,
   const [data, setData] = useState([])
   const [downloading, setDownloading] = useState(false)
   const classes = useStyles()
+  const [order, setOrder] = useState()
+  const [orderBy, setorderBy] = useState()
+  const [orderDirection, setorderDirection] = useState('asc')
```
Comment (dsgarcia8) [inline src/views/ComparisonView/ComparisonView.jsx:74]: rename `setorderDirection` to `setOrderDirection`

**Review by dsgarcia8 — CHANGES_REQUESTED** [review]
Nice job, please check if the order should be applied in the other columns like sector, vertical, revenue, etc.

File: src/views/ComparisonView/ComparisonView.jsx [hunk 2]
```diff
@@ -61,6 +69,10 @@ export function ComparisonView ({ companyComparison, peersComparison, isLoading,
   const [data, setData] = useState([])
   const [downloading, setDownloading] = useState(false)
   const classes = useStyles()
+  const [order, setOrder] = useState()
+  const [orderBy, setorderBy] = useState()
```
[the remark this answers is not in the document]
Comment (FreddyJR1995) [reply src/views/ComparisonView/ComparisonView.jsx]: It was renamed

File: src/views/ComparisonView/ComparisonView.jsx [hunk 3]
```diff
@@ -61,6 +69,10 @@ export function ComparisonView ({ companyComparison, peersComparison, isLoading,
...
+  const [orderDirection, setorderDirection] = useState('asc')
+  const [valueToOrderBy, setvalueToOrderBy] = useState('name')
```
[the remark this answers is not in the document]
Comment (FreddyJR1995) [reply src/views/ComparisonView/ComparisonView.jsx]: It was also renamed

**Review by dsgarcia8 — APPROVED** [review]

**Pull request merged**